apps/web/src/pages/brand/tabs/v2/BrandingForm.tsx

```diff
@@ -0,0 +1,232 @@
+                        })}
+                      >
+                        {!field.value ? (
+                          <Stack gap="4px" direction="column" align="center" justify="center" flexGrow={1}>
+                            <ActionButton onClick={() => dropzoneRef.current?.()} Icon={IconOutlineFileUpload} />
+                            <Text color={colors.B70}>Upload</Text>
+                          </Stack>
+                        ) : (
+                          <img
+                            data-test-id="logo-image-wrapper"
+                            src={field.value}
+                            className={css({
+                              minHeight: '6.25rem',
+                              minWidth: '6.25rem',
+                              objectFit: 'contain',
+                            })}
+                            alt="avatar"
+                          />
+                        )}
+                      </div>
+                    </Dropzone>
+                  </div>
+                )}
+                control={control}
+                name="brandLogoUrl"
+              />
+            </Stack>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <ColorInput label="Brand color" data-test-id="brand-color-picker" disallowInput={false} {...field} />
+                )}
+                control={control}
+                name="brandColorValue"
+              />
+            </div>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <Select
+                    label="In-App font family"
+                    placeholder="Select a font family"
+                    data={[
+                      'inherit',
+                      'Fira Code',
+                      'Roboto',
+                      'Montserrat',
+                      'Open Sans',
+                      'Lato',
+                      'Nunito',
+                      'Oswald',
+                      'Raleway',
+                    ]}
+                    data-test-id="font-family-selector"
+                    {...field}
+                  />
+                )}
+                control={control}
+                name="fontFamilyValue"
+              />
+            </div>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <ColorInput
+                    label="In-App font color"
+                    data-test-id="font-color-picker"
+                    disallowInput={false}
+                    {...field}
+                  />
+                )}
+                control={control}
+                name="fontColorValue"
+              />
+            </div>
+          </Stack>
+
+          <div
+            className={css({
+              textAlign: 'right',
+              marginTop: '60px',
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ We should reconsider this value and confirm with Nikolay‏

Reply:
This is copied from the old file